Tests/SonarScanner.MSBuild.PostProcessor.Tests/TargetsUninstallerTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿/*
+ * SonarScanner for MSBuild
+ * Copyright (C) 2016-2021 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.IO;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarScanner.MSBuild.PostProcessor;
+using TestUtilities;
+
+namespace MSBuild.SonarQube.Internal.PostProcess.Tests
+{
+    [TestClass]
+    public class TargetsUninstallerTests
+    {
+        public TestContext TestContext { get; set; }
+
+        [TestMethod]
+        public void Ctor_Argument_Check()
+        {
+            Action action = () => new TargetsUninstaller(null);
+            action.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("logger");
+        }
+
+        [TestMethod]
+        public void DeleteFileWhenPresent()
+        {
+            var context = new UninstallContext(TestContext, true);
+            context.UninstallTargets();
+            File.Exists(context.TargetsFilePath).Should().BeFalse();
+            context.Logger.AssertDebugLogged("Uninstalling target: " + context.TargetsFilePath);
+        }
+
+        [TestMethod]
+        public void Log_MissingFile()
+        {
+            var context = new UninstallContext(TestContext, false);
+            context.UninstallTargets();
+            context.Logger.AssertDebugLogged(context.BinDir + @"\targets\SonarQube.Integration.targets does not exist");
```

Comment:
The other tests use `context.TargetsFilePath` instead of constructing the path again. Could it be used here?